src/test/blk_pool_lock/blk_pool_lock.c

```diff
@@ -184,9 +184,9 @@ main(int argc, char *argv[])
 
 	if (argc == 2) {
 		test_reopen(argv[1]);
-		test_open_in_different_process(argv[1], 0);
+		test_open_in_different_process(argv, 0);
 		for (int i = 1; i < 100000; i *= 2)
-			test_open_in_different_process(argv[1], i);
+			test_open_in_different_process(argv, i);
```

Comment:
Instead of turning the variable named "path" to an array, add a second argument to those functions.